src/modules/vtol_att_control/tiltrotor.cpp

```diff
@@ -266,35 +267,91 @@ void Tiltrotor::update_external_state()
 
 }
 
- /**
-* Prepare message to acutators with data from the attitude controllers.
+/**
+* Write data to actuator output topic.
 */
-void Tiltrotor::fill_att_control_output()
+void Tiltrotor::fill_actuator_outputs()
 {
-	_actuators_out_0->control[0] = _actuators_mc_in->control[0] * _roll_weight_mc;	// roll
-	_actuators_out_0->control[1] = _actuators_mc_in->control[1] * _roll_weight_mc;	// pitch
-	_actuators_out_0->control[2] = _actuators_mc_in->control[2] * _roll_weight_mc;	// yaw
+	switch(_vtol_schedule.flight_mode) {
```

Comment:
What about setting up an enum for the indexes to help self document the code and less error prone? Then we'd have:
```
_actuators_out_0->control[ROLL] = _actuators_mc_in->control[ROLL];
_actuators_out_0->control[PITCH] = _actuators_mc_in->control[PITCH];
...
```
While we are at it, any reason we can't change ```_actuators_out_0 / _actuators_out_1``` to ```_actuators_out_mc / _actuators_out_fw``` or at least ```_actuators_out_main / _actuators_out_aux```?